src/components/CGroupTable.vue

```diff
@@ -73,7 +74,7 @@ export default {
     this.$store.commit('setCurrentPath', this.$route.path);
     this.$store.dispatch('startRefresh');
   },
-  beforeDestroy() {
+  beforeCreate() {
```

Comment:
why change this ?

Reply:
@antoinerabany  it was an unexpecting behaviour in the request, I change it as a part of the solution of the issue of twice requesting and to make it more semantic, since the initialization should happen before an element is mounted